services/web-server/src/main.js

```diff
@@ -56,7 +56,7 @@ const load = loader(
 
         return new Client({
           monitor: monitor.childMonitor('pulse-client'),
-          namespace: cfg.pulse.username,
+          namespace: cfg.pulse.namespace,
```

Comment:
Other services hard-code this value. Should we do the same and the remove `namespace` from the config file?

Reply:
I think that will make it harder to run this service locally against a pulse service.  If that's not a problem, then we could hard-code.  But I think this change will be sufficient to work in production.